apis/v1alpha1/gatewayclass_types.go

```diff
@@ -93,9 +93,8 @@ type GatewayClassSpec struct {
 	//
 	// Support: Core
 	//
-	// +optional
 	// +kubebuilder:default={onlySameNamespace:true}
-	AllowedRouteNamespaces RouteNamespaces `json:"allowedRouteNamespaces,omitempty"`
+	AllowedRouteNamespaces *RouteNamespaces `json:"allowedRouteNamespaces,omitempty"`
```

Comment:
Same as https://github.com/kubernetes-sigs/service-apis/pull/336/files#r492273748

Reply:
Field is removed now, thanks!